libs/src/sdk/api/chats/ChatsApi.ts

```diff
@@ -255,10 +345,22 @@ export class ChatsApi
     return (await response.json()) as TypedCommsResponse<string[]>
   }
 
-  public async getBlockees() {
+  /**
+   * Gets the user ids the current user has blocked
+   * @param requestParameters.currentUserId the user to act on behalf of
+   * @returns
+   */
+  public async getBlockees(requestParameters: ChatGetBlockersRequest) {
```

Comment:
is there an easy way to make `requestParameters` optional from a schema perspective @sliptype so that this isn't a breaking change? I could add a `?` here but would `parseRequestParameters` need to be conditionally run then too?

Reply:
You could have a top level `z.optional()` wrapping the entire schema but that won't really help in this case because you still won't be able to make `requestParameters` an optional arg. Seems like conditionally calling parse would be the best approach. Or we could make `requestParameters` an optional arg on `parseRequestParameters` and return an empty object?

Reply:
Like this? https://github.com/AudiusProject/audius-protocol/pull/5415/commits/cb69656c1268f01638f7b533840104d849d6bdbd

Reply:
Niiiice, I like that